cli/cmd/version/version.go

```diff
@@ -0,0 +1,50 @@
+package version
+
+import (
+	"fmt"
+	"strings"
+)
+
+var (
+	// The git commit that was compiled. These will be filled in by the compiler.
+	GitCommit   string
+	GitDescribe string
+
+	// The main version number that is being run at the moment.
+	//
+	// Version must conform to the format expected by
+	// github.com/hashicorp/go-version for tests to work.
+	Version = "0.0.1"
+
+	// A pre-release marker for the version. If this is "" (empty string)
+	// then it means that it is a final release. Otherwise, this is a pre-release
+	// such as "dev" (in development), "beta", "rc1", etc.
+	VersionPrerelease = ""
+)
+
+// GetHumanVersion composes the parts of the version in a way that's suitable
+// for displaying to humans.
+func GetHumanVersion() string {
```

Comment:
This file is copied from: https://github.com/hashicorp/consul-k8s/blob/main/control-plane/version/version.go, and similar to https://github.com/hashicorp/consul/blob/main/version/version.go which I probably should have referenced in the file itself, and it is only ever modified by release CI. For that reason, I think it's ok if it doesn't have tests, as it's not tech debt in that a human would need to modify this file. WDYT?

Reply:
Hm, totally fine for the initial release. Might be worth rethinking how we are copying and pasting code throughout. It could get unmaintainable. For now, not a problem.